### PR TITLE
Fix google maps inconsistent loading

### PR DESCRIPTION
**Pull request by georgeschena**

### CODE REVIEW

File: app/views/admin/govwifi_map/index.html.erb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 <%= link_to "Back to organisations", admin_organisations_path, class: "govuk-back-link" %>
  <h1 class="govuk-heading-l">GovWifi Map of Locations</h1>
-  <script async defer src="https://maps.googleapis.com/maps/api/js?key=<%= ENV['GOOGLE_MAPS_PUBLIC_API_KEY'] %>&callback=initMap"></script>
 
   <div class="map"></div>
    <script>
-     function initMap() {
+   window.initMap = function(){
```
Comment (emileswarts) [inline app/views/admin/govwifi_map/index.html.erb]: Indenting is a bit off if you look at the script tag start.
No space after closing )

File: app/views/admin/govwifi_map/index.html.erb [hunk 2]
```diff
@@ -25,3 +24,4 @@
        }
      }
    </script>
+<script src="https://maps.googleapis.com/maps/api/js?key=<%= ENV['GOOGLE_MAPS_PUBLIC_API_KEY'] %>&callback=initMap" async defer></script>
```
Comment (emileswarts) [inline app/views/admin/govwifi_map/index.html.erb]: This script tag should be aligned with the script tag above it?

File: app/views/admin/govwifi_map/index.html.erb (hunk 1 above)
Comment (cybojenix) [reply app/views/admin/govwifi_map/index.html.erb]: Can you define as `window.initMap = function initMap()` please, which will help with debugging purposes.

with `window.initMap = function()`, it shows up in traces as an anonymous function

Comment (cybojenix) [reply app/views/admin/govwifi_map/index.html.erb]: Also unsure if it needs attaching to `window`. The problem will have been from the fetched script being cached, which loads before we reach this `script` block defining `initMap`.
Moving the `script` block to above the async script should fix the issue

**Pull request merged**